### DISCUSSION TITLE
JOB

### DISCUSSION THREAD

**Discussion by rudazy** [community/community#58632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Can you guys make Github easier to look for Jobs And also get seen base on what is written on our profile?

Coz I'm a new user but everything seems so difficult to understand

**Comment by ghostinhershell** [comment]
Hi @rudazy, we're happy you're here 😀

Thanks for posting in the GitHub Community, @rudazy! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

Also, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by gooddavvy** [comment]
I joined last year but I need a job at GitHub too @rudazy

**Reply by rudazy** [reply]
Hopefully the team told me I've been added to the category so I hope to get an email soon from any company interested
